### DISCUSSION TITLE
Change default server name from 'ps7' to 'master'

### DISCUSSION THREAD

**Discussion by djravine** [djravine/idlecombos#42] [Announcements]
It appears that the API Server default `ps7` is no longer responding.
It might be under maintenance or decommissioned.

I have changed the default to `ps20` which seems to fix the data loading issue.
**_FYI:_** `ps17`, `ps20` also appears to work fine.
**_FYI:_** `master` may have restrictions on the server side and may cause 24 hour soft bans.

**_NOTE:_** You may have to manually change this setting yourself, as it is saved in the user settings for this app and don't forget to click the `SAVE SETTINGS` button.

![2023-02-14 10 19 32](https://user-images.githubusercontent.com/155802/218605684-619e22ec-9808-4236-b80b-35a2c4751e5f.png)

**Comment by raxele123** [comment]
Byteglow says I'm on ps20

**Comment by breffasst** [comment]
I switched to master then redeemed 20-25 Electrum Chest codes and attempted to open some silver chests which failed.
Afterwards I attempted to login to the Steam Client and was notified of a 24 hour suspension :( 

`{"success":false,"failure_reason":"This call can not be made to this server.","error_code":99,"recovery_options":"refresh","actions":[],"processing_time":"0.02442","memory_usage":"3.25 mb","apc_stats":{"gets":719,"gets_time":"0.00000","sets":0,"sets_time":"0.00000"},"db_stats":{"2":false,"1":false,"26":false}}`

**Reply by enonethreezed** [reply]
Same issue here 30 minutes ago. Same message, same restriction.

**Reply by djravine** [reply]
Same for me also, Issues regarding EVENT CHESTS also https://github.com/djravine/idlecombos/issues/45
No restriction on EPIC GAMES client.

`{"success":false,"failure_reason":"Outdated instance id","error_code":-1,"recovery_options":"","processing_time":"0.00319","memory_usage":"4 mb","apc_stats":{"gets":0,"gets_time":"0.00000","sets":0,"sets_time":"0.00000"},"db_stats":{"23":false}}`

[Byteglow](https://ic.byteglow.com/) also appears to be having issues downloading user data.
![2023-02-14 09 50 29](https://user-images.githubusercontent.com/155802/218600506-5b638909-4edd-45c7-be1f-65b0548cf6b0.png)

**Reply by djravine** [reply]
I have switched from `master` to `ps20` for testing.

**Reply by djravine** [reply]
`ps20` appears to be working.
I have opened 50 silver chests without problem.
I also opened 100 silver chests without problem.

**Reply by djravine** [reply]
Release [v3.48](https://github.com/djravine/idlecombos/releases/tag/v3.48) now sets the default server name to `ps20`
You will need to manually change if you have already saved settings previously.
